terminology_bot/requirements.txt

```diff
@@ -0,0 +1,4 @@
+python_telegram_bot==11.1.0
+psycopg2==2.7.6.1
+SQLAlchemy==1.2.14
+telegram==0.0.1
```

Comment:
telegram==0.0.1 - bot.py doesn't work with this line.
Fails with:
Traceback (most recent call last):
  File "bot.py", line 4, in <module>
    from telegram import (ReplyKeyboardMarkup, ReplyKeyboardRemove)
ImportError: cannot import name 'ReplyKeyboardMarkup'